### ISSUE TITLE
Error on vagrant deployment sync: "Can't set cookie dm_task_set_cookie failed"

### ISSUE THREAD

**Issue by rmohr** [kubevirt/kubevirt#241]
Recently I see this error a lot, when building new containers and  trying to start them on our CentOS7 vagrant VMs:

```
Error response from daemon: {\"message\":\"devmapper: Error activating devmapper device for 'd113d5e10d347e4abf83ed26229a350abbfb21c133365b30e6939dde83bf192d-init': devicemapper: Can't set cookie dm_task_set_cookie failed\"}"
```
The containers get stuck in the `ContainerCreating` phase:

```
haproxy-3749063686-s0pmv                  0/2       ContainerCreating   0          5m
iscsi-demo-target-tgtd-2918391489-994zw   0/1       ContainerCreating   0          5m
kubevirt-cockpit-demo-1842943600-l8qs6    1/1       Running             0          5m
libvirt-m543v                             2/2       Running             0          5m
libvirt-qsfk8                             0/2       ContainerCreating   0          5m
spice-proxy-2868258710-pzw96              0/1       ContainerCreating   0          5m
virt-api-3813486938-m84h6                 0/1       ContainerCreating   0          5m
virt-controller-1975339297-bdbcv          0/1       ContainerCreating   0          5m
virt-handler-12txz                        1/1       Running             0          5m
virt-handler-bf4t4                        0/1       ContainerCreating   0          5m
virt-manifest-2726702614-hxh3g            0/2       ContainerCreating   0          5m

``` 

Does anyone else experience that? It might be related to using docker through CRI ...

Only a restart of the VMs solves the problem.

**Comment by fabiand** [comment]
No, I have not seen it before.

**Comment by berrange** [comment]
Yes, I tried vagrant on Friday, and saw this failure. THe initial vagrant deployment worked fine, but as soon as I tried to re-deploy changed code it crashed & burned with this message. Even restarting the VM didn't always help. There are some docker bug reports suggesting it is neccessary to increase the number of semaphores but that didn't help for me.

Personally I'd prefer to switch the vagrant setup to use Fedora instead of RHEL so we are running on a more modern platform. We're not really getting the benefits of stable RHEL base, since its pulling in a virt party repo for updates virt pieces which replaces lots of RHEL packages.
